crates/ruma-events/CHANGELOG.md

```diff
@@ -27,6 +27,11 @@ Breaking changes:
   using `MediaPreviewConfigEventContent` for the stable version and `MediaPreviewConfigEventContent`
   for the unstable one, both are only implemented as `GlobalAccountData` events as of right now.
    
+Bug fixes:
+
```

Comment:
Next time, feel free to do the suggestion and apply it yourself, so as to help move forward faster :wink: 

```suggestion
```